src/assets/i18n/cs.json5

```diff
@@ -4936,7 +4934,7 @@
   "iiif.page.doi": "Trvalý odkaz: ",
 
   // "iiif.page.issue": "Issue: ",
-  "iiif.page.issue": "Číslo: ",
+  "iiif.page.issue": "Problém:",
```

Comment:
Tady mi chybí kontext (kde se to v tom IIIF ukazuje), nicméně v es.json je překlad, kterej odpovídá tomu původnímu českému `Číslo: `. Ono to bude asi spíš něco jako číslo vydání, než nahlašte problém, ne?